src/amruta/Coding_Exam_1/Program_1.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Given an array of integers nums and an integer target, 
+ * return indexes of the two numbers such that they add up to the target.
+
+Input: nums = [2,11,15,7,4], target = 9
+
+Output: [0,3]
+
+Output: Because nums[0] + nums[3] == 9, we return [0, 3].
+ */
+
+
+package amruta.Coding_Exam_1;
+
+public class Program_1 {
+
+	int target=9;
+
+	void getindex(int[] arr){
+		for(int i=0; i < arr.length; i++) {
+			for (int j=1; j< arr.length; j++){
```

Comment:
Inner for loop should from i+1, think on this, if you want to connect on Skype then feel free to message.

Take reference of this code
Kangan - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/987/files
Rupali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/991/files
Dipali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1003/files